<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyBindingsWindow</class>
 <widget class="QDialog" name="KeyBindingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>370</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Silkscreen</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Key Bindings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/dm_icon.ico</normaloff>../img/dm_icon.ico</iconset>
  </property>
  <widget class="QPushButton" name="setUpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setLeftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setRightButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setDownButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setSelectButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setStartButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setBButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setAButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set A</string>
   </property>
  </widget>
  <widget class="QLabel" name="startKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="aKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="selectKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>O</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="leftKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="upKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="downKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rightKeyLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bindStatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>[select a button to bind]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
